packages/AreaChart/src/area-chart.scss

```diff
@@ -18,10 +30,35 @@
   --ids-area-chart-empty-label-text-padding: #{tokens.$space-3};
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-area-chart-font-family: var(--hop-body-xs-font-family);
+  --ids-area-chart-font-size: var(--hop-body-xs-font-size);
+  --ids-area-chart-font-weight: var(--hop-body-xs-font-weight);
+  --ids-area-chart-color: var(--hop-neutral-text-weak);
+  --ids-area-chart-grid-color: var(--hop-neutral-border-weakest);
+  --ids-area-chart-axis-color: var(--hop-neutral-border-weak);
+  --ids-area-chart-dot-color: #BFDCA9; /* Missing semantic token */
+  --ids-area-chart-dot-stroke-color: #ffffff; /* Missing semantic token */
+  --ids-area-chart-line-color: url("#lineGradient");
```

Comment:
I would use a more unique theme as if many SVG are in the page this could be problematic.